emsdk.py

```diff
@@ -1525,7 +1528,7 @@ def build_binaryen_tool(tool):
   return success
 
 
-def download_and_unzip(zipfile, dest_dir, download_even_if_exists=False,
+def download_and_unzip(zipfile, dest_dir, unpack_even_if_exists=False,
```

Comment:
Maybe not worth renaming this variable.  Its used to avoid both download and unpacking, so I'm not sure one name is better than than the other.    Maybe we could avoid the verb and just call it `ignore_existing` ?

Reply:
I just kept the old name, which cuts this patch size a lot. Since the `download_even_if_exists` was always set to `True` originally, this is now really a `unpack_even_if_exists`, as the download now just depends on `KEEP_DOWNLOADS`, but the `unpack_even_if_exists` now isn't always True anymore.